tests/api.v2/routes/experiment.test.js

```diff
@@ -3,6 +3,7 @@ const request = require('supertest');
 const expressLoader = require('../../../src/loaders/express');
 
 jest.mock('../../../src/api.v2/controllers/experimentController');
+jest.mock('../../../src/utils/authMiddlewares');
```

Comment:
Are you planning to write tests for `patchExperiment` and `updateSamplePosition`?

Reply:
Added tests